clippy_lints/src/utils/paths.rs

```diff
@@ -8,8 +8,8 @@ pub const ANY_TRAIT: [&str; 3] = ["std", "any", "Any"];
 pub const ARC_PTR_EQ: [&str; 4] = ["alloc", "sync", "Arc", "ptr_eq"];
 pub const ASMUT_TRAIT: [&str; 3] = ["core", "convert", "AsMut"];
 pub const ASREF_TRAIT: [&str; 3] = ["core", "convert", "AsRef"];
-pub const BEGIN_PANIC: [&str; 3] = ["std", "panicking", "begin_panic"];
-pub const BEGIN_PANIC_FMT: [&str; 3] = ["std", "panicking", "begin_panic_fmt"];
+pub(super) const BEGIN_PANIC: [&str; 3] = ["std", "panicking", "begin_panic"];
+pub(super) const BEGIN_PANIC_FMT: [&str; 3] = ["std", "panicking", "begin_panic_fmt"];
```

Comment:
Please remove the `(super)` for consistency.

Reply:
I added the `(super)` limitation to ensure that people only use these paths through `match_panic_call` and `match_panic_def_id`; the paths shouldn't be used directly because then much more things need to be kept in sync. I think leaving them as only `pub(super)` makes sense, but I can remove it if you really want.

Reply:
Hm. Good point. Let's leave the super.